sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -786,13 +789,13 @@ def set_http_headers(self, file_attributes="none",  # type: Union[str, NTFSAttri
             The file system attributes for files and directories.
             If not set, indicates preservation of existing values.
             Here is an example for when the var type is str: 'Temporary|Archive'
-        :type file_attributes: str or :class:`~azure.storage.fileshare.NTFSAttributes`
+        :type file_attributes: str or ~azure.storage.fileshare.NTFSAttributes
```

Comment:
Omitting :class: here still renders this properly? I'm sure you checked just checking 😄

Reply:
Yup, `~azure.storage.fileshare.NTFSAtrributes` renders properly! `pylint` complains about the fact there's `:class:` in type docstrings, so that's what we removed. Here is what the rendered docs look like:
![Screenshot 2024-04-09 at 22 31 17](https://github.com/Azure/azure-sdk-for-python/assets/162184229/001fe53a-e3fe-4f15-b388-c8cba0847e64)